Formula/git.rb

```diff
@@ -22,6 +22,14 @@ class Git < Formula
   depends_on "gettext"
   depends_on "pcre2"
 
+  on_linux do
+    depends_on "curl"
+    depends_on "expat"
+    depends_on "linux-headers"
+    depends_on "openssl@1.1"
+    depends_on "zlib"
+  end
```

Comment:
```suggestion
  uses_from_macos "curl"
  uses_from_macos "expat"
  uses_from_macos "zlib"

  on_linux do
    depends_on "linux-headers"
    depends_on "openssl@1.1"
  end
```

Reply:
Actually, I think `uses_from_macos "openssl@1.1"` is also allowed, so I can change that as well.